core/common/src/main/java/alluxio/AlluxioURI.java

```diff
@@ -228,6 +228,25 @@ public boolean containsWildcard() {
     return mUri.getPath().contains(WILDCARD);
   }
 
+  /**
+   * Gets the host of the {@link AlluxioURI}.
+   *
+   * @return the host, null if it does not have one
+   */
+  @Nullable
+  public String getHost() {
+    return mUri.getHost();
+  }
+
+  /**
+   *Gets the port of the {@link AlluxioURI}.
+   *
+   * @return the port, -1 if it does not have one
+   */
+  public int getPort() {
+    return mUri.getPort();
+  }
+
```

Comment:
I don't think these two new methods are necessary.
see my comments in `CephFSUnderFileSystem.createInstance`